mosquitto/hooks/run

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -xe
+
+exec 2>&1
+
+exec mosquitto -c {{pkg.svc_config_path}}/mosquitto.conf
```

Comment:
Since this is such a simple run hook would you mind moving it to the plan with `pkg_svc_run="mosquitto -c $pkg_svc_config_path/mosquitto.conf"`

Reply:
Done :)